### PR TITLE
Kirstie weekly update | 13 to 19 August 

### PR DESCRIPTION
**Pull request by KirstieJane**
My updates for the last week.

Let me know what you think!

(I’m not sure why I have Georgia’s update in here - I’ll fix when I’m not on my phone!)

### CODE REVIEW

File: Kirstie-Whitaker/Weekly-meetings.md [hunk 1]
```diff
@@ -18,6 +18,61 @@ Archives for 2018 can be found in [Weekly-meetings_2018.md](weekly-meetings-2018
 
 ---
 
+### Date: 19 August 2019
+
+#### Who did you help this week?
+
+* Went over GitHub pull requests w YH
```
Comment (minnieho1115) [inline Kirstie-Whitaker/Weekly-meetings.md]: Thank Kirstie for teaching me a lot of useful Github knowledge.:heart:

File: Kirstie-Whitaker/Weekly-meetings.md [hunk 2]
```diff
@@ -18,6 +18,61 @@ Archives for 2018 can be found in [Weekly-meetings_2018.md](weekly-meetings-2018
...
+* Ran PhD student office hours which I think are helpful - someone even came :laughing:
+
+#### Who helped you this week?
+
+* SDC made some awesome explaining GitHub slides from our conversation which are so freaking useful.
+  * http://doi.org/10.5281/zenodo.3369466
```
Comment (minnieho1115) [inline Kirstie-Whitaker/Weekly-meetings.md]: I have read it.It is simple,clear and absolutely great!

File: Kirstie-Whitaker/Weekly-meetings.md [hunk 3]
```diff
@@ -18,6 +18,61 @@ Archives for 2018 can be found in [Weekly-meetings_2018.md](weekly-meetings-2018
...
+* Suggested demographic information to collect to improve moinitoring of applications for winter school
+* Interview w journalist re: Binder :sparkling_heart:
+* Released updated version for Raincloud plots!
+  Whoops that that took so long :grimacing:
+  https://github.com/RainCloudPlots/RainCloudPlots
+* Re-synced BEP001 with the main specification, ran mini-hack call
+* Chaired GSOC meeting & reviewed great final reports, excited for them to appear on the blog this week!
+* Chaired tedana monthly meeting
+* Great blog posts in the process of being reviewed by the lab members!!
+
```
Comment (minnieho1115) [inline Kirstie-Whitaker/Weekly-meetings.md]: It is a fruitful week!:bowtie:

File: Kirstie-Whitaker/Weekly-meetings.md [hunk 4]
```diff
@@ -18,6 +18,61 @@ Archives for 2018 can be found in [Weekly-meetings_2018.md](weekly-meetings-2018
...
+#### What did you struggle with?
+
+* I had two days blocked out for writing and they both disappeared :sob:.
+  I need to keep those goals focused.
+* What are manageable and interesting questions in natural stimuli fMRI analyses!?!
```
Comment (minnieho1115) [inline Kirstie-Whitaker/Weekly-meetings.md]: >+What are manageable and interesting questions in natural stimuli fMRI analyses!?!
I am also interested in~

File: Kirstie-Whitaker/Weekly-meetings.md [hunk 5]
```diff
@@ -18,6 +18,61 @@ Archives for 2018 can be found in [Weekly-meetings_2018.md](weekly-meetings-2018
...
+* Excited for MS's visit to Turing & getting *The Turing Way* ramped back up!
+* Get another of RY's PRs merged, and maybe one more??
+* Keep email inbox under control?
+* Review updated manuscript for RR
+* I worked late every night this week - need to do more exercise :muscle:
+
+#### Where do you need help?
+
+* I feel kinda ok - need more time in the week obviously, but not feeling outrageously overwhelmed...
+* Always helpful to see folks working across projects without me :smile:
```
Comment (minnieho1115) [inline Kirstie-Whitaker/Weekly-meetings.md]: The teamwork of our Whitaker's lab is great!I love our lab!

File: Kirstie-Whitaker/abstracts_bios_2019.md [hunk 6]
```diff
@@ -310,6 +312,68 @@ You can find more information at her lab website: [whitakerlab.github.io](https:
...
+Dr Whitaker uses magnetic resonance imaging to study child and adolescent brain development and participatory citizen science to educate non-autistic people about how they can better support autistic friends and colleagues.
+She is the lead developer of [*The Turing Way*](https://github.com/alan-turing-institute/the-turing-way), an openly developed educational resource to enable more reproducible data science.
+Kirstie is a passionate advocate for making science "open for all" by promoting equity and inclusion for people from diverse backgrounds, and by changing the academic incentive structure to reward collaborative working.
+She is the chair of the Turing Institute's Ethics Advisory Group, a Fulbright scholarship alumna and was a 2016/17 [Mozilla Fellow for Science](https://science.mozilla.org/programs/fellowships/fellows).
+Kirstie was named, with her collaborator Petra Vertes, as a [2016 Global Thinker](https://gt.foreignpolicy.com/2016/profile/petra-vertes-and-kirstie-whitaker) by Foreign Policy magazine.
+You can find more information at her lab website: [whitakerlab.github.io](https://whitakerlab.github.io/).
+
+**Slides**: Figshare link to slides (if applicable)
+
+
```
Comment (minnieho1115) [inline Kirstie-Whitaker/abstracts_bios_2019.md]: Terrific job!Let's make 'The Turing Way'  better and better!:heart:

File: Kirstie-Whitaker/Weekly-meetings.md (hunk 3 above)
Comment (sparkler0323) [reply Kirstie-Whitaker/Weekly-meetings.md]: Plenty of work and the Raincloud plots really look great!

File: Kirstie-Whitaker/Weekly-meetings.md (hunk 2 above)
Comment (sparkler0323) [reply Kirstie-Whitaker/Weekly-meetings.md]: Nice work 👍

**Comment by KirstieJane** [comment]
Thanks for the reviews @sparkler0323 & @minnieho1115! Merged now!

**Pull request merged**